testing/trino-testing/src/main/java/io/trino/testing/AbstractTestDistributedQueries.java

```diff
@@ -69,7 +69,7 @@
  * @see AbstractTestIntegrationSmokeTest
  */
 public abstract class AbstractTestDistributedQueries
```

Comment:
This has to be renamed now, surely the word `Queries` has to disappear.

Reply:
I do not see how it does have to be renamed